package.json

```diff
@@ -13,7 +13,7 @@
   },
   "homepage": "https://github.com/aerogear/graphql-metadata/#readme",
   "main": "dist/index.js",
-  "types": "types/index.d.ts",
+  "types": "dist/index.d.ts",
```

Comment:
This will be related to the typescript version we use, but since we have one locked in repo there is no issue

Reply:
Background? I don't know what you mean.

Reply:
ignore. Default folder for types was changed in typescript versions. But this is ancient info